mobile/src/androidTest/java/org/openhab/habdroid/ui/BasicWidgetTest.java

```diff
@@ -45,48 +49,25 @@
 public class BasicWidgetTest {
 
     @Rule
-    public ActivityTestRule<OpenHABMainActivity> mActivityTestRule = new ActivityTestRule<>(OpenHABMainActivity.class);
+    public ActivityTestRule<OpenHABMainActivity> mActivityTestRule = new ActivityTestRule<>
+            (OpenHABMainActivity.class, true, false);
 
     private IdlingResource mProgressbarIdlingResource;
 
-    @Test
-    public void openHABMainActivityTest2() throws InterruptedException {
-        // click next
-        ViewInteraction appCompatImageButton = onView(
-                allOf(withId(R.id.next),
-                        childAtPosition(
-                                allOf(withId(R.id.bottomContainer),
-                                        childAtPosition(
-                                                withId(R.id.bottom),
-                                                1)),
-                                3),
-                        isDisplayed()));
-        appCompatImageButton.perform(click());
-
-        // click next
-        ViewInteraction appCompatImageButton2 = onView(
-                allOf(withId(R.id.next),
-                        childAtPosition(
-                                allOf(withId(R.id.bottomContainer),
-                                        childAtPosition(
-                                                withId(R.id.bottom),
-                                                1)),
-                                3),
-                        isDisplayed()));
-        appCompatImageButton2.perform(click());
+    @Before
+    public void setup() {
+        PreferenceManager
+                .getDefaultSharedPreferences(InstrumentationRegistry.getTargetContext())
+                .edit()
+                .putString(Constants.PREFERENCE_SITEMAP, "")
+                .putBoolean(Constants.PREFERENCE_FIRST_START, false)
+                .commit();
 
-        // close intro
-        ViewInteraction appCompatButton = onView(
-                allOf(withId(R.id.done), withText("DONE"),
-                        childAtPosition(
-                                allOf(withId(R.id.bottomContainer),
-                                        childAtPosition(
-                                                withId(R.id.bottom),
-                                                1)),
-                                4),
-                        isDisplayed()));
-        appCompatButton.perform(click());
+        mActivityTestRule.launchActivity(null);
+    }
 
+    @Test
+    public void openHABMainActivityTest2() throws InterruptedException {
```

Comment:
The "2" was introduced by me, when I capture it via Android Studio. Can you remove it?

Reply:
+1 :)